enhancements/olm-crd-conversion-webhooks.md

```diff
@@ -0,0 +1,102 @@
+---
+title: olm-crd-conversion-webhook-support
+authors:
+  - "@sdhaliwa"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-06-10
+last-updated: 2020-06-15
+status: implementable
+---
+
+# OLM CRD Conversion Webhook Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement covers the necessary steps to add the capability for OLM to support [conversion](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/#webhook-conversion) webhooks.
+
+## Motivation
+
+Many Operators are shipping conversion webhooks to deliver conversion of an object from one version to another. The CustomResourceDefinition API supports a versions field that is used to support multiple versions of custom resources that have been developed. Versions can have different schemas and a conversion webhook is used to convert custom resources between versions.
+OLM needs to support operators with crd conversion webhooks and help manage and rotate corresponding certificates.
+
+### Goals
+
+- Operator authors can include crd conversion webhooks in their bundle as a part of a CSV.
+- Operator author will not need to manage cert rotation for webhooks shipped with their operators.
+
+## Proposal
+
+Operator authors will be able to:
+
+- Ship their operator with a CRD conversion webhook without implementing logic to create or manage cert information as OLM will take care of that requirement instead.
+
+- Define webhook within CSV 
```

Comment:
This already exists. We want authors to identify which webhook is a conversion webhook.